### PR TITLE
Double Halloween - END

### PR DESCRIPTION
**Pull request by EthanStraub**
- Added Sherman back in
- Nerfed thicker than blood when hunting enemies. Now only gives out 75% of remaining slime (down from 100%), while the rest is sent to the district
- Added in Slimy Persuits, a candy store in New New Yonkers
- Halloween costumes can now only be adorned on a full moon
- Added in Bicarbonate enemies, which are more powerful, more vulnerable to attacks, and completely immune to Bicarbonate Rainstorms
- An intense tremor has caved in certain parts of the city. Evacuations to Juvie's Row have been made.
- Removed Double Halloween
- Nerfed !boo buffering (it's now only one response instead of two)

### CODE REVIEW

File: ewapt.py [hunk 1]
```diff
@@ -983,240 +983,6 @@ def check(str):
 		response = "One door at a time, please."
 		return await ewutils.send_message(cmd.client, cmd.message.author, ewutils.formatMessage(cmd.message.author, response))
 
-# TODO: Remove after Double Halloween
-async def trickortreat(cmd = None):
```
Comment (magicksRampage) [inline ewapt.py:987]: should we really delete all this code? I think it'd make sense to keep it around for next year. just remove the commands from the command map. this goes for all the double halloween commands

File: ewcfg.py [hunk 2]
```diff
@@ -9657,59 +9608,26 @@ def atf_hooves(ctn=None):
...
+			"sp",
+			"slimypersuits",
+			"slimeypersuits",
+			"candystore",
+			"candyshop",
+			"candy store",
+			"candy shop",
+		],
+		str_name="Slimy Persuits",
+		str_desc="It's a candy Shop [DEBUG]",
```
Comment (magicksRampage) [inline ewcfg.py]: add a real description?

File: ewcosmeticitem.py [hunk 3]
```diff
@@ -88,6 +88,9 @@ def __init__(
 
 				if i.item_props.get("adorned") == 'true':
 					already_adorned = True
+				if i.item_props.get("context") == 'costume':
```
Comment (magicksRampage) [inline ewcosmeticitem.py]: shouldn't we also automatically dedorn them when the full moon passes?

**Review by magicksRampage — CHANGES_REQUESTED** [review]

File: ewapt.py (hunk 1 above)
Comment (EthanStraub) [reply ewapt.py:987]: Yeah, that sounds more reasonable. I forgot that a command is essentially removed from the game if it's not on the command map anyways.

File: client.py [hunk 4]
```diff
@@ -830,6 +831,8 @@
 
 					# Advance the time and potentially change weather.
 					market_data.clock += 1
+					if not ewutils.check_fursuit_active(market_data.id_server):
+						ewcosmeticitem.dedorn_all_costumes()
```
Comment (magicksRampage) [inline client.py]: market_data hasn't been persisted yet, so this will be an hour late

File: ewcfg.py [hunk 5]
```diff
@@ -9657,59 +9620,26 @@ def atf_hooves(ctn=None):
...
+			"sp",
+			"slimypersuits",
+			"slimeypersuits",
+			"candystore",
+			"candyshop",
+			"candy store",
+			"candy shop",
+		],
+		str_name="Slimy Persuits",
+		str_desc="It's a vintage style candy store, and on top of that an ice-cream parlour. Sugary delicacies line the displays, giving the whole place an inviting prescense and sweet scent. One of the signs on the walls tells of their signature product, the Slime Sours. Apprently they're made almost entirely by hand, and a lot of the other products in the store seem to fit that bill as well. In post-apocalyptic hellscape like NLCakaNM, it seems some traditions have still survived.",
```
Comment (magicksRampage) [inline ewcfg.py]: typos: prescense, NLCakaNM, In a post-apocalyptic

File: ewcosmeticitem.py [hunk 6]
```diff
@@ -251,3 +257,18 @@ def __init__(
...
+
+
+def dedorn_all_costumes():
+	costumes = ewutils.execute_sql_query("SELECT id_item FROM items_prop WHERE name = 'context' AND value = 'costume' AND id_item IN (SELECT id_item FROM items_prop WHERE name = 'adorned' AND value = 'true')")
+	costume_count = 0
+
+	for costume_id in costumes:
+		costume_item = EwItem(id_item=costume_id)
+		
+		costume_item.item_props['adorned'] = 'false'
```
Comment (magicksRampage) [inline ewcosmeticitem.py:269]: also dedorn them from slimeoids

File: ewcosmeticitem.py (hunk 3 above)
Comment (magicksRampage) [reply ewcosmeticitem.py]: I forgot we also need to check this for slimeoids

**Review by magicksRampage — CHANGES_REQUESTED** [review]

**Pull request merged**